er3t/util/util.py

```diff
@@ -404,6 +406,72 @@ def get_doy_tag(date, day_interval=8):
     return doy_tag
 
 
+def get_lon2lat2(lon1, lat1, bearing, distance, use_great_circle=True):
```

Comment:
How about rename this function to `cal_lonlat_by_dist` so it is more self-explanatory? Maybe array operation is possible (e.g., input array of `lon1`, `lat1` and output array `lon2`, `lat2`) by replacing `math` functions with `numpy` functions.